### PR TITLE
feat(TextLink): new component

### PR DESCRIPTION
**Pull request by GaroGabrielyan**
### Chromatic Build Summary 
**Build URL:** https://www.chromatic.com/build?appId=66daee7859a843dd87bfb3d7&number=90 
**Storybook URL:** https://66daee7859a843dd87bfb3d7-hwcoqcnvws.chromatic.com/

### CODE REVIEW

**Review by NairaMisakyan — APPROVED** [review]

File: src/lib/atoms/TextLink/TextLink.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        className={classNames(`textLink textLink_color_${appearance}`, {
+            textLink_underline: underline,
+            textLink_disabled: disabled
+        })}
+        href={href}
+        onFocus={onFocus}
+        onClick={onClick}
+    >
+        {Icon && Icon}
+        <span className="textLink__text">{children}</span>
```
Comment (GaroGabrielyan) [inline src/lib/atoms/TextLink/TextLink.tsx:86]: Need to add truncated case.

File: src/lib/atoms/TextLink/TextLink.test.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React, { FocusEvent, MouseEvent } from 'react';
+import { ReactWrapper, mount } from 'enzyme';
+
+// Components
+import TextLink, { ITextLinkProps } from './index';
+
+describe('TextLink', () => {
```
[the remark this answers is not in the document]
Comment (NarekArshakyan) [reply src/lib/atoms/TextLink/TextLink.test.tsx:7]: Test for Icons, i will add when Icon component become ready

File: src/lib/atoms/TextLink/TextLink.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+const meta: Meta<typeof TextLink> = {
+    title: 'Atoms/TextLink',
+    component: TextLink,
+    argTypes: {
+        appearance: args({ control: 'select', ...propCategory.options }),
+        disabled: args({ control: 'boolean', ...propCategory.functionality }),
+        href: args({ control: 'text', ...propCategory.functionality }),
+        Icon: args({ control: 'false', ...propCategory.appearance }),
+        onClick: args({ control: 'false', ...propCategory.functionality }),
+        onFocus: args({ control: 'false', ...propCategory.functionality }),
```
[the remark this answers is not in the document]
Comment (NarekArshakyan) [reply src/lib/atoms/TextLink/TextLink.stories.tsx:19]: I think you mean onClick . I answer in this question in comment upside.

[the remark this answers is not in the document]
Comment (NarekArshakyan) [reply src/lib/atoms/TextLink/TextLink.stories.tsx:19]: I think functionality is more suitable for it.

File: src/lib/atoms/TextLink/TextLink.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    rel?: 'none' | 'nofollow';
+    /**
+     * When underline is supplied, we override the underline style internally managed by the component.
+     */
+    underline?: boolean;
+    /**
+     * Indicates the browsing context where an href will be opened:
+     * Possible values: <code>self | blank</code>
+     */
+    target?: 'self' | 'blank';
```
[the remark this answers is not in the document]
Comment (NarekArshakyan) [reply src/lib/atoms/TextLink/TextLink.tsx:30]: https://betconstruct.atlassian.net/browse/CORE-1507

File: src/lib/atoms/TextLink/TextLink.tsx (hunk 1 above)
Comment (NarekArshakyan) [reply src/lib/atoms/TextLink/TextLink.tsx:86]: I think this is for the layout team

**Pull request merged**